doc/source/suse-ecp/ose-configure.rst

```diff
@@ -120,12 +116,12 @@ For example:
 
 
 Configure the VIP that will be used for Airship UCP service endpoints
---------------------------------------------------------------------------
+---------------------------------------------------------------------
```

Comment:
arent we duplicating this info? Most of this section is already explained in the generic docs so only things that are different from a normal deployment should be mentioned here

Reply:
recommend addressing later